google-scripts/process-form-submission/__tests__/setUniqueIdOnSubmission.test.ts

```diff
@@ -0,0 +1,125 @@
+import { setUniqueIdOnSubmission } from "../setUniqueIdOnSubmission";
+import { MockSpreadsheetApp } from "../__mocks__/google_mocks";
+
+describe("#setUniqueIdOnSubmission", () => {
+    let mockFormData: { [key: string]: string[] };
```

Comment:
Some of the formatting in this file, and others, looks a bit off. Can we check if we're running `prettier` on these files? If not, my recommendation is usually:

* Have `prettier` run on save in your editor
* Have a `yarn lint` job that validates the formatting is all correct as per `prettier`'s expectations (you should be able to run `prettier --check` to do this in our CI pipeline)

Reply:
Done in 2dd6e1d